lib/px/utils/pxclient.lua

```diff
@@ -242,6 +242,8 @@ function M.load(config_file)
 
         if ngx.var.cookie__pxvid then
             vid = ngx.var.cookie__pxvid
+        elseif ngx.var.cookie_pxvid then
+            vid = ngx.var.cookie_pxvid
         elseif ngx.var.cookie_vid then
```

Comment:
this is not needed anymore, if I understand correctly.
@dimapx ?

Reply:
@dimapx  said when i checked with him on slack that  we need to check for both _pxvid and pxvid (nodejs does the same)

Reply:
but not `vid`...

Reply:
@pxjohnny @yaronpx `pxvid` replaces `vid`, such that the check should be against `_pxvid` and `pxvid`. no need to check for a `vid` cookie key. 
In node it is exactly as it should be - https://github.com/PerimeterX/perimeterx-node-core/blob/master/lib/pxproxy.js#L108.